Casks/0-ad.rb

```diff
@@ -4,10 +4,14 @@
 
   url "https://releases.wildfiregames.com/0ad-#{version}-osx64.dmg",
       verified: "releases.wildfiregames.com/"
-  appcast "https://play0ad.com/download/mac/"
   name "0 A.D."
   desc "Real-time strategy game"
   homepage "https://play0ad.com/"
 
+  livecheck do
+    url "https://play0ad.com/download/mac/"
+    regex(/0ad-(.*?)-osx64\.dmg/i)
```

Comment:
```suggestion
    regex(/href=.*?0ad[._-]v?(.*?)-osx64\.dmg/i)
```

* Since we're matching a file on an HTML page, this restricts matching to an `href` attribute.
* Uses the generic `[._-]` between the software name and numeric version (this is something we commonly do with filename regexes).

Past that, this cask uses an alpha version, so I've left the capture group loose.